src/channel.ts

```diff
@@ -326,14 +328,25 @@ export class Channel<
         ReactionType,
         UserType
       >;
-      offset?: number;
       query?: string;
     } = {},
   ) {
     // Return a list of channels
-    const payload = {
-      filter_conditions: { cid: this.cid },
-      ...options,
+    const { sort, ...optionsWithoutSort } = { ...options };
```

Comment:
So sort here is throwing a TS error, this is because only one of the 2 search options types has sort on it.

Reply:
Don't think you need to spread options here.